projects/step-core/src/lib/components/toast/toast.component.scss

```diff
@@ -0,0 +1,36 @@
+.toast {
```

Comment:
Please use `rem` instead of `px`
Use colors variables defined in `core-variables`